nevergrad/functions/ml/test_mlfunctionlib.py

```diff
@@ -12,30 +12,54 @@
 
 
 def test_ml_tuning() -> None:
+    # Testing a decision tree with only depth as a parameter.
+    np.random.seed(17)
```

Comment:
to prove it's working well, you should go for `func.rng.seed(17)` instead

Reply:
Done!

Reply:
Done, thx!